polyfill/lib/ecmascript.mjs

```diff
@@ -5170,6 +5170,9 @@ export const ES = ObjectAssign({}, ES2022, {
     }
     return quotient;
   },
+  BigIntIfAvailable: (wrapper) => {
```

Comment:
Do we really need to support this in the non-production polyfill?

Reply:
At the last time I discussed this, BigInt support in Safari was recent enough (September 2020) that it seemed best to keep the bigInteger library even just for the purposes of the browser playground. I'd say it's probably not necessary to support it any longer, but I also don't see it as a priority to spend time removing it.